cumulusci.rb

```diff
@@ -3,8 +3,8 @@ class Cumulusci < Formula
 
   desc "Python framework for building automation for Salesforce projects"
   homepage "https://github.com/SFDO-Tooling/CumulusCI"
-  url "https://files.pythonhosted.org/packages/ec/bd/2116bd208a7d2b4930ef9bbe4d83317bca494e29636208364ae57d81bf75/cumulusci-3.61.1.tar.gz"
-  sha256 "0cab41c2d77e86f7bf1fd37f6b0dfa962c5a94d675c25507afa2c2c0422f189d"
+  url "https://files.pythonhosted.org/packages/12/90/97a8ec175a0591b713134502bb88c7f05ed22682976e3620fb831e524b3f/cumulusci-3.62.0.tar.gz"
```

Comment:
It looks like the hash here does pickup that it's version 3.62. You should be safe to merge.

Reply:
Awesome I saw that too! My question is why the title and branch are wrong?